mage_ai/settings/sso.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+# Okta settings
+
+OKTA_DOMAIN_URL = os.getenv('OKTA_DOMAIN_URL')
+OKTA_CLIENT_ID = os.getenv('OKTA_CLIENT_ID')
+OKTA_CLIENT_SECRET = os.getenv('OKTA_CLIENT_SECRET')
+
+# Google settings
```

Comment:
Could you add more comments? like these settings are used to set up authentication.